concepticondata/conceptlists/Sawka-2019-201.tsv

```diff
@@ -0,0 +1,202 @@
+Sawka-2019-201-21	21	hair	Body Part	1040	HAIR
+Sawka-2019-201-22	22	head	Body Part	1256	HEAD
+Sawka-2019-201-23	23	heart	Body Part	1223	HEART
+Sawka-2019-201-24	24	jawbone	Body Part	798	JAW
+Sawka-2019-201-25	25	knee	Body Part	1371	KNEE
+Sawka-2019-201-26	26	leg	Body Part	1297	LEG
+Sawka-2019-201-27	27	liver	Body Part	1224	LIVER
+Sawka-2019-201-28	28	lung	Body Part	688	LUNG
+Sawka-2019-201-29	29	mouth	Body Part	674	MOUTH
+Sawka-2019-201-30	30	neck	Body Part	1333	NECK
+Sawka-2019-201-31	31	nose	Body Part	1221	NOSE
+Sawka-2019-201-32	32	skin	Body Part	763	SKIN
+Sawka-2019-201-33	33	sore	Body Part	51	SORE
+Sawka-2019-201-34	34	tail	Body Part	1220	TAIL
+Sawka-2019-201-35	35	tongue	Body Part	1205	TONGUE
+Sawka-2019-201-36	36	tooth	Body Part	1380	TOOTH
+Sawka-2019-201-37	37	voice	Body Part	1363	VOICE
+Sawka-2019-201-38	38	wing	Body Part	1257	WING
+Sawka-2019-201-39	39	one	number	1493	ONE
+Sawka-2019-201-40	40	two	number	1498	TWO
+Sawka-2019-201-41	41	three	number	492	THREE
+Sawka-2019-201-42	42	four	number	1500	FOUR
+Sawka-2019-201-43	43	five	number	493	FIVE
+Sawka-2019-201-44	44	six	number	1703	SIX
+Sawka-2019-201-45	45	seven	number	1704	SEVEN
+Sawka-2019-201-46	46	eight	number	1705	EIGHT
+Sawka-2019-201-47	47	nine	number	1483	NINE
+Sawka-2019-201-48	48	ten	number	1515	TEN
+Sawka-2019-201-49	49	few	number	1242	FEW
+Sawka-2019-201-50	50	many	number	1198	MANY
+Sawka-2019-201-51	51	bee	animal	665	BEE
+Sawka-2019-201-52	52	bird	animal	937	BIRD
+Sawka-2019-201-53	53	buffalo	animal	934	BUFFALO
+Sawka-2019-201-54	54	cat	animal	1208	CAT
+Sawka-2019-201-55	55	crocodile	animal	1857	CROCODILE
+Sawka-2019-201-56	56	dog	animal	2009	DOG
+Sawka-2019-201-57	57	elephant	animal	1290	ELEPHANT
+Sawka-2019-201-58	58	fish	animal	227	FISH
+Sawka-2019-201-59	59	goat	animal	1502	GOAT
+Sawka-2019-201-60	60	head louse	animal	310	HEAD LOUSE
+Sawka-2019-201-61	61	hippo	animal	3145	HIPPOPOTAMUS
+Sawka-2019-201-62	62	lion	animal	1386	LION
+Sawka-2019-201-63	63	rhino	animal	2591	RHINOCEROS
+Sawka-2019-201-64	64	snake	animal	730	SNAKE
+Sawka-2019-201-65	65	spider	animal	843	SPIDER
+Sawka-2019-201-66	66	wild animal	animal	958	WILD ANIMAL
+Sawka-2019-201-67	67	bull	animal	1008	BULL
+Sawka-2019-201-68	68	calf	animal	1989	CALF
+Sawka-2019-201-69	69	cow	animal	1007	COW
+Sawka-2019-201-70	70	chicken	animal	1318	CHICKEN
+Sawka-2019-201-71	71	cock	animal	1511	ROOSTER
+Sawka-2019-201-72	72	arrow	thing	977	ARROW
+Sawka-2019-201-73	73	ash	thing	646	ASH
+Sawka-2019-201-74	74	axe	thing	677	AXE
+Sawka-2019-201-75	75	bark (of tree)	thing	1204	BARK
+Sawka-2019-201-76	76	burden	thing	1679	LOAD
+Sawka-2019-201-77	77	chair	thing	1476	CHAIR
+Sawka-2019-201-78	78	cloud	thing	1489	CLOUD
+Sawka-2019-201-79	79	cooking pot	thing	3653	COOKING POT
+Sawka-2019-201-80	80	darkness	thing	11	DARKNESS
+Sawka-2019-201-81	81	daytime	thing	1225	DAY (NOT NIGHT)
+Sawka-2019-201-82	82	dew	thing	1977	DEW
+Sawka-2019-201-83	83	dust	thing	2	DUST
+Sawka-2019-201-84	84	earth	thing		
```

Comment:
Potentially 1228? But you probably had a reason for not mapping that?

Reply:
I' map it to EARTH (SOIL)

Reply:
Just because they did not specify a meaning and there are several options for earth, but since you and Kristina agree I will map it to 1228.